src/ui/zcl_abapgit_services_repo.clas.abap

```diff
@@ -135,6 +142,22 @@ CLASS ZCL_ABAPGIT_SERVICES_REPO IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD import_zip.
```

Comment:
guess this moves the responsibility from somewhere else? it can be done in a separate PR

Reply:
same as above - I can unroll to `WHEN zif_abapgit_definitions=>c_action-zip_import` above if you want, but the benefit is just reducing number of changed objects in PR. The code will be the same.